app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -343,56 +334,69 @@ class Contact extends React.Component {
+                        .map(value => {
+                          return { label: customPreferredContactTimeOptions[`${value}`], value };
                         })
-                      }
-                      styles={preferredContactTimeSelectStyling}
-                      theme={theme => bootstrapSelectTheme(theme, 'lg')}
-                    />
-                    <div className="mt-3">
-                      <span className="font-weight-bold">Morning: </span>
-                      <span className="font-weight-light">Between 8:00 and 12:00 in monitoree&apos;s timezone</span>
-                      <br />
-                      <span className="font-weight-bold">Afternoon: </span>
-                      <span className="font-weight-light">Between 12:00 and 16:00 in monitoree&apos;s timezone</span>
-                      <br />
-                      <span className="font-weight-bold">Evening: </span>
-                      <span className="font-weight-light">Between 16:00 and 20:00 in monitoree&apos;s timezone</span>
-                    </div>
-                    <Form.Control.Feedback className="d-block" type="invalid">
-                      {this.state.errors['preferred_contact_time']}
-                    </Form.Control.Feedback>
-                  </Form.Group>
-                )}
+                    )}
+                    onChange={e =>
+                      this.handleChange({
+                        target: { id: 'preferred_contact_time', value: e.value },
+                        currentTarget: { id: 'preferred_contact_time' },
+                      })
+                    }
+                    styles={preferredContactTimeSelectStyling}
+                    theme={theme => bootstrapSelectTheme(theme, 'lg')}
+                  />
+
+                  <div className="mt-3">
+                    <span className="font-weight-bold">Morning: </span>
+                    <span className="font-weight-light">Between 8:00 and 12:00 in monitoree&apos;s timezone</span>
+                    <br />
+                    <span className="font-weight-bold">Afternoon: </span>
+                    <span className="font-weight-light">Between 12:00 and 16:00 in monitoree&apos;s timezone</span>
+                    <br />
+                    <span className="font-weight-bold">Evening: </span>
+                    <span className="font-weight-light">Between 16:00 and 20:00 in monitoree&apos;s timezone</span>
+                  </div>
+                  <Form.Control.Feedback className="d-block" type="invalid">
+                    {this.state.errors['preferred_contact_time']}
+                  </Form.Control.Feedback>
+                  {![null, ''].includes(this.state.current.patient.preferred_contact_time) && (
+                    <React.Fragment>
+                      {[null, ''].includes(this.state.current.patient.preferred_contact_method) &&
+                        this.renderWarningBanner(
```

Comment:
I don't love the location of these warning banners, I think it would look better above the contact time descriptions

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1106/commits/ce6a5a2e27cd5120050ddef6601ebcc7abbcaab5

Reply:
looks much better!